hazelcast-client/src/main/java/com/hazelcast/client/map/impl/nearcache/ClientHeapNearCache.java

```diff
@@ -194,7 +194,6 @@ public Object get(K key) {
         fireTtlCleanup();
         NearCacheRecord record = cache.get(key);
         if (record != null) {
-            record.access();
```

Comment:
This updates the last access timestamp, so an already expired entry will not be evicted.